web_widget_numeric_step/static/src/js/numeric_step.js

```diff
@@ -198,10 +213,11 @@ odoo.define('web_widget_numeric_step.field', function (require) {
         /**
          * Sanitize user input value
          *
-         * @param {ChangeEvent} ev
+         * @override
          */
         _onChange: function (ev) {
             ev.target.value = this._sanitizeNumberValue(ev.target.value);
+            this._super.apply(this, arguments);
```

Comment:
Possibly not strictly needed, but this is the common pattern:

```suggestion
            return this._super.apply(this, arguments);
```